sdk/cdn/Azure.ResourceManager.Cdn/api/Azure.ResourceManager.Cdn.netstandard2.0.cs

```diff
@@ -4266,7 +4266,7 @@ public partial class WafPolicyManagedRuleSet
     public partial class WafPolicySettings
     {
         public WafPolicySettings() { }
-        public string DefaultCustomBlockResponseBody { get { throw null; } set { } }
+        public System.BinaryData DefaultCustomBlockResponseBody { get { throw null; } set { } }
```

Comment:
Do we have a new rule that the property ends with `Body` need to become `BinaryData`?

Reply:
no, manual is ok